server/api/config/config.js

```diff
@@ -1 +1,16 @@
-export const secret = 'worldisfullofdevelopers';
+const dotenv = require('dotenv');
+
+const env = process.env.NODE_ENV;
+dotenv.config();
+
+const dev = {
+	db: process.env.DATABASE_URL,
+};
+const test = {
+	db: process.env.DATABASE_TEST_URL,
+};
+const config = {
+	dev,
+	test,
+};
+module.exports = config[env];
```

Comment:
Why can't you use `ES6` with `export` or `export default`?

Reply:
I have changed it.